content/docs/introducing-jsx.md

```diff
@@ -6,31 +6,35 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Kijk eens naar de volgende variabele toewijzing:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Deze rare tag syntax is noch een string, noch HTML.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Dit heet JSX en het is een syntax uitbreiding voor JavaScript.
+Wij raden aan om het samen met React te gebruiken om te beschrijven hoe de UI er uit moet zien.
+JSX doet je misschien denken aan een template taal, maar het heeft alle voordelen van JavaScript.
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX produceert React "elementen". In het [volgende hoofdstuk](/docs/rendering-elements.html) gaan we zien hoe ze in de DOM gerenderd worden.
+Hieronder vind je de basis van JSX die je nodig hebt om te beginnen.
 
-### Why JSX? {#why-jsx}
+### Waarom JSX? {#why-jsx}
 
-React embraces the fact that rendering logic is inherently coupled with other UI logic: how events are handled, how the state changes over time, and how the data is prepared for display.
+React omarmt het feit dat de logica voor het renderen vanzelfsprekend gekoppeld is met andere UI logica: hoe events worden afgehandeld, hoe de state veranderd na een tijdje en hoe de data wordt voorbereid voor weergave.
```

Comment:
I think it's best to use the English terms for all/most of the programming terms. This will make it easier to connect the information here with information found elsewhere, I think.

Reply:
Just wanted to confirm, all good then :)